mzLib/FlashLFQ/ChromatographicPeak.cs

```diff
@@ -40,27 +44,85 @@ public ChromatographicPeak(Identification id, bool isMbrPeak, SpectraFileInfo fi
             IsotopicEnvelopes = new List<IsotopicEnvelope>();
             IsMbrPeak = isMbrPeak;
             SpectraFileInfo = fileInfo;
+            RandomRt = randomRt;
         }
 
-        public ChromatographicPeak(Identification id, bool isMbrPeak, SpectraFileInfo fileInfo, double predictedRetentionTime) :
-            this(id, isMbrPeak, fileInfo)
+        public bool Equals(ChromatographicPeak peak)
         {
-            PredictedRetentionTime = predictedRetentionTime;
+            return SpectraFileInfo.Equals(peak.SpectraFileInfo) 
+                && Identifications.First().ModifiedSequence.Equals(peak.Identifications.First().ModifiedSequence)
+                && ApexRetentionTime == peak.ApexRetentionTime;
         }
 
+
         public IsotopicEnvelope Apex { get; private set; }
         public List<Identification> Identifications { get; private set; }
         public int NumChargeStatesObserved { get; private set; }
         public int NumIdentificationsByBaseSeq { get; private set; }
         public int NumIdentificationsByFullSeq { get; private set; }
         public double MassError { get; private set; }
```

Comment:
Most of your new properties are not directly covered by tests. Is this worrisome?